bpdm-gate/src/main/kotlin/org/eclipse/tractusx/bpdm/gate/config/ClientsConfig.kt

```diff
@@ -51,6 +53,22 @@ class OrchestratorClientConfig {
         return OrchestrationApiClientImpl { webClientBuilder(url).build() }
     }
 
+    @Bean
+    @ConditionalOnProperty(
+        value = ["bpdm.gate-security.pool-security-enabled"],
+        havingValue = "false",
+        matchIfMissing = true
+    )
+    fun poolClientNoAuth(poolConfigProperties: PoolConfigProperties): PoolApiClient {
```

Comment:
We should add the auth profile for the pool since the pool on dev and int are using auth and by that the connection will fail

Reply:
True, currently there is no pool client configuration in case the security is enabled. Should be introduced